get_externals.lua

```diff
@@ -35,85 +33,100 @@ local externals =
 	},
 }
 
-local zipFullName = FindFileInPath("7z.exe");
+local zipFullName = "7z"

```

Comment:
as this will probably only work on linux, you can also use `7zr` here which is installed by default on ubuntu while `7z` is not